monkey/infection_monkey/network/relay/utils.py

```diff
@@ -90,8 +89,7 @@ def send_remove_from_waitlist_control_message_to_relays(servers: Iterable[str]):
 
 
 def _send_remove_from_waitlist_control_message_to_relay(server: str):
-    ip, port = address_to_ip_port(server)
-    server_address = SocketAddress(IPv4Address(ip), int(port))
+    server_address = SocketAddress(*address_to_ip_port(server))
```

Comment:
don't you need to provide kwargs to SocketAddress, since it's a pydantic model?

Reply:
Use this once it's merged: https://github.com/guardicore/monkey/pull/2350/files#diff-69582fdb56607bef080bc859a5d7007b039dbbad57e4abae63eda498626b96b6

Reply:
This will need to be addressed